prelude-si/tilt.bzl

```diff
@@ -124,12 +124,16 @@ def _invoke_tilt(ctx: AnalysisContext, subcmd: str) -> list[[DefaultInfo, RunInf
         ctx.attrs.tiltfile,
     )
 
+    RunInfo(["rm", "-rf", "/tmp/si-logs"])
```

Comment:
Nothing like an `rm -rf` to wake you up in the morning

Reply:
This one is tactical